src/snovault/schema_graph.py

```diff
@@ -84,8 +92,17 @@ def schema_dot(request):
 
 @view_config(route_name='graph_svg', request_method='GET')
 def schema_svg(request):
-    dot = digraph(request.registry[TYPES].by_item_type, request.params.getall('exclude'))
-    p = Popen(['dot', '-Tsvg'], stdin=PIPE, stdout=PIPE, stderr=PIPE)
-    svg, err = p.communicate(dot.encode('utf-8'))
-    assert p.returncode == 0, err.decode('utf-8')
-    return Response(svg, content_type='image/svg+xml', charset='utf-8')
+    try:
+        return_code = subprocess_call(['which', 'dot'])
```

Comment:
Having the application check its dependencies seems a bit odd in this context. We should control the dependencies through install scripts and check functionality with view tests.

Reply:
well, the view is calling the command.  Shouldn't it handle the scenario when the command doesn't exist?